tests/acceptance/test_organization_events_v2.py

```diff
@@ -10,8 +10,10 @@
 
 FEATURE_NAMES = ("organizations:events-v2", "organizations:discover-v2-query-builder")
 
-all_view = "field=title&field=event.type&field=project&field=user&field=timestamp&alias=title&alias=type&alias=project&alias=user&alias=time&name=All+Events&sort=-timestamp&tag=event.type&tag=release&tag=project.name&tag=user.email&tag=user.ip&tag=environment"
-error_view = "field=title&alias=error&field=count%28id%29&alias=events&field=count_unique%28user%29&alias=users&field=project&alias=project&field=last_seen&alias=last+seen&name=Errors&query=event.type%3Aerror&sort=-last_seen&sort=-title&tag=error.type&tag=project.name"
+all_events_query = "alias=title&alias=type&alias=project&alias=user&alias=time&field=title&field=event.type&field=project&field=user&field=timestamp&name=All+Events&sort=-timestamp&tag=event.type&tag=release&tag=project.name&tag=user.email&tag=user.ip&tag=environment"
```

Comment:
You and I might conflict on these lines.

Reply:
@markstory what do you mean by that? 🤔

Reply:
In hindsight I'll end up with the merge conflict :smile: